app/controllers/RepresentativeDanImportVATController.scala

```diff
@@ -105,4 +118,13 @@ class RepresentativeDanImportVATController @Inject()(identify: IdentifierAction,
     }
   }
 
+  private[controllers] def sameAccountNumber(dutyAccountNumber: String,
+                                             vatAccountNumber: String,
+                                             userAnswers: UserAnswers) = {
+    lazy val dutyAccountNumber: String = userAnswers.get(DefermentAccountPage).getOrElse("")
+    dutyAccountNumber.exists(
+      accountNumber => dutyAccountNumber == vatAccountNumber
+    )
```

Comment:
we don't need `getOrElse` here either`
```suggestion
    val dutyAccountNumber = userAnswers.get(DefermentAccountPage)
    dutyAccountNumber.exists(_ == vatAccountNumber)
```